### DISCUSSION TITLE
Hello and Trying to use GIT PUSH...

### DISCUSSION THREAD

**Discussion by silver2row** [community/community#39225] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello,

I am trying to use ` git push `. It seems that when I use it on the master branch, things work. Then, I switch repos. online via ` cloning ` and receive errors w/ the same passkey phrase from the ` settings/developer settings/personal access token `.

So, I basically have it working w/ one repo. and then switch repos. and the same ` access token ` no longer works w/ the newer ` cloned ` repo.

Is this normal? Should I need to restart a new ` access token ` when pushing to new repos. on github?

Seth

**Comment by DSDmark** [comment, marked as answer]
Hi @silver2row,

Check first, your token have expired or not.
Go there and check [**GitHub token**](https://github.com/settings/tokens)

And try that ```git push origin master``` instead of ```git push```.

**Reply by silver2row** [reply]
Hello @DSDmark ,

Okay...I will test ` git push origin master ` instead. Thank you for the update.

Seth

**Reply by silver2row** [reply]
That did it. I must have been readin' the wrong docs.

Seth

**Reply by silver2row** [reply]
Also...for some reason, one of my repos. has a ` main ` instead of ` master ` branch. 

I think this is an older repo. branch naming scheme. 

Seth

P.S. I think this is when ` git push ` used to work out-of-the-box.

**Reply by DSDmark** [reply]
Ya, if you want, then you can set your default **branch**. Inside setting and page section.
Here is shortcut link if you want to set default branch:- https://github.com/settings/repositories

![2022-11-16_19-50](https://user-images.githubusercontent.com/85744816/202173673-67bbec0d-1bee-4222-a332-1efcbec26d60.png)
